cmd/application_scale.go

```diff
@@ -0,0 +1,79 @@
+package cmd
+
+import (
+	"fmt"
+	"os"
+	"strconv"
+	"strings"
+
+	"github.com/civo/civogo"
+	"github.com/civo/cli/config"
+	"github.com/civo/cli/utility"
+	"github.com/spf13/cobra"
+)
+
+func ScaleCmd(cmd *cobra.Command, args []string) {
```

Comment:
nit: I don't think this needs to be exported. However, if it does , linter won't be happy as you haven't provided a comment on top of it.

Reply:
Done.